RecoLocalTracker/SiPixelClusterizer/plugins/gpuClustering.h

```diff
@@ -88,6 +111,9 @@ namespace gpuClustering {
       jmax[k] = msize;
 
     __syncthreads();
+    if (threadIdx.x==0 && hist.fullSpill()) printf("histo overflow in det %d\n",thisModuleId);
```

Comment:
can you `#ifdef` this on some debug macro ?

Reply:
this is aLogWarning equivalent...